src/game/scenes/mechlab/lab_menu_customize.c

```diff
@@ -62,14 +62,14 @@ int calculate_trade_value(sd_pilot *pilot) {
     return trade_value * 0.85;
 }
 
-static const int32_t arm_leg_value[10] = {0, 9, 34, 94, 197, 351, 606, 1034, 1675, 2700};
-static const int32_t armor_value[10] = {0, 22, 86, 235, 492, 876, 1516, 2584, 4186, 6748};
-static const int32_t stun_value[10] = {0, 13, 51, 141, 294, 526, 910, 1551, 2511, 4463};
+static const int32_t arm_leg_value[10] = {0, 10, 40, 111, 231, 412, 713, 1216, 1969, 3175};
+static const int32_t armor_value[10] = {0, 25, 100, 277, 578, 1030, 1784, 3040, 4924, 7939};
+static const int32_t stun_value[10] = {0, 15, 60, 166, 347, 618, 1070, 1824, 2954, 4763};
```

Comment:
Probably need to find some formula to make these numbers for other bots too here, these are for jaguar.